ocaml/runtime4/caml/compact.h

```diff
@@ -2,9 +2,9 @@
 /*                                                                        */
```

Comment:
(The diff of this file is unexpected...)

Reply:
This must have been a bad automatic merge.  This file now matches `flambda-backend/main`.